gcc/common/config/riscv/riscv-common.cc

```diff
@@ -1462,6 +1463,7 @@ static const riscv_ext_flag_table_t riscv_ext_flag_table[] =
   {"xtheadmemidx",  &gcc_options::x_riscv_xthead_subext, MASK_XTHEADMEMIDX},
   {"xtheadmempair", &gcc_options::x_riscv_xthead_subext, MASK_XTHEADMEMPAIR},
   {"xtheadsync",    &gcc_options::x_riscv_xthead_subext, MASK_XTHEADSYNC},
+  {"xcvmac",        &gcc_options::x_riscv_xcv_flags, MASK_XCVMAC},
```

Comment:
Move above `xtheadba` to keep in alphabetical order.

Reply:
Changed